examples/dreambooth/train_dreambooth_lora.py

```diff
@@ -839,6 +839,11 @@ def main(args):
         else:
             raise ValueError("xformers is not available. Make sure it is installed correctly")
 
+    if args.gradient_checkpointing:
+        unet.enable_gradient_checkpointing()
+        if args.train_text_encoder:
+            text_encoder.gradient_checkpointing_enable()
```

Comment:
`enable_gradient_checkpointing()` and `gradient_checkpointing_enable()` will raise eyebrows but no biggie